app/ante/setup.go

```diff
@@ -0,0 +1,188 @@
+package ante
+
+import (
+	"errors"
+	"strconv"
+
+	sdkmath "cosmossdk.io/math"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
Some of this functionality has been moved to `sdkerrors.io/errors` I believe
```suggestion
	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
	sdkerrors "cosmossdk.io/errors"
```